From dc28810f06e2dc46eb6c1beef0cfba088ed1f68c Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Mon, 4 Apr 2016 00:07:36 +0200
Subject: [PATCH] do not require system bus for running test suite

Do not require system bus when running the test suite as this makes it
harder to run the test suite in a restricted environment like on
buildds. As the DBus connection seems to be unused, simply remove it
altogether.

Bug: https://bugzilla.gnome.org/show_bug.cgi?id=764562
---
 gcr/test-system-prompt.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/gcr/test-system-prompt.c b/gcr/test-system-prompt.c
index 3016252..703a7cb 100644
--- a/gcr/test-system-prompt.c
+++ b/gcr/test-system-prompt.c
@@ -697,7 +697,6 @@ static void
 test_watch_cancels (Test *test,
                     gconstpointer unused)
 {
-	GDBusConnection *connection;
 	GcrPrompt *prompt;
 	GcrPrompt *prompt2;
 	GError *error = NULL;
@@ -707,9 +706,6 @@ test_watch_cancels (Test *test,
 	gcr_mock_prompter_set_delay_msec (3000);
 	gcr_mock_prompter_expect_password_ok ("booo", NULL);
 
-	connection = g_bus_get_sync (G_BUS_TYPE_SYSTEM, NULL, &error);
-	g_assert_no_error (error);
-
 	/* This should happen immediately */
 	prompt = gcr_system_prompt_open_for_prompter (test->prompter_name, 0, NULL, &error);
 	g_assert_no_error (error);
@@ -744,7 +740,6 @@ test_watch_cancels (Test *test,
 	g_object_unref (prompt);
 	g_object_unref (pair.result1);
 	g_object_unref (pair.result2);
-	g_object_unref (connection);
 }
 
 int
-- 
2.8.0.rc3

